### PR TITLE
#116: Aviation recipe is missing process.sql

### PR DESCRIPTION
**Pull request by ybyzek**
All recipes need a `process.sql` for 1-click in next phase

### CODE REVIEW

File: docs/customer-360/aviation/process.sql [hunk 1]
```diff
@@ -0,0 +1,86 @@
+SET 'auto.offset.reset' = 'earliest';
+
+CREATE TABLE customers (ID             INT     PRIMARY KEY
+                       , NAME           VARCHAR
```
Comment (mjsax) [inline docs/customer-360/aviation/process.sql:4]: Formatting seems broken?

File: docs/customer-360/aviation/process.sql [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                       , ADDRESS        VARCHAR
+                       , EMAIL          VARCHAR
+                       , PHONE          VARCHAR
+                       , LOYALTY_STATUS VARCHAR)
+              WITH (KAFKA_TOPIC='customers'
+                   , FORMAT='AVRO'
+                   , PARTITIONS=6);
+
+CREATE TABLE flights (ID               INT     PRIMARY KEY
```
Comment (mjsax) [inline docs/customer-360/aviation/process.sql:13]: as above

File: docs/customer-360/aviation/process.sql [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                       , ORIGIN        VARCHAR
+                       , DESTINATION   VARCHAR
+                       , CODE          VARCHAR
+                       , SCHEDULED_DEP TIMESTAMP
+                       , SCHEDULED_ARR TIMESTAMP)
+              WITH (KAFKA_TOPIC='flights'
+                   , FORMAT='AVRO'
+                   , PARTITIONS=6);
+
+CREATE TABLE bookings (ID            INT     PRIMARY KEY
```
Comment (mjsax) [inline docs/customer-360/aviation/process.sql:23]: same here

File: docs/customer-360/aviation/process.sql [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  SELECT CB.*, F.*
+  FROM   customer_bookings CB
+          INNER JOIN flights F
+              ON CB.FLIGHT_ID=F.ID;
+
+CREATE STREAM cf_stream WITH (KAFKA_TOPIC='customer_flights', FORMAT='AVRO');
+
+CREATE STREAM cf_rekey WITH (KAFKA_TOPIC='cf_rekey') AS 
+  SELECT F_ID                 AS FLIGHT_ID
+        , CB_C_ID             AS CUSTOMER_ID
```
Comment (mjsax) [inline docs/customer-360/aviation/process.sql:47]: and here

File: docs/customer-360/aviation/process.sql [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+        , F_SCHEDULED_ARR     AS FLIGHT_SCHEDULED_ARR
+  FROM cf_stream
+  PARTITION BY F_ID;
+
+CREATE TABLE customer_flights_rekeyed 
+  (FLIGHT_ID INT PRIMARY KEY) 
+  WITH (KAFKA_TOPIC='cf_rekey', FORMAT='AVRO');
+
+CREATE STREAM flight_updates (ID          INT KEY
+                            , FLIGHT_ID   INT
```
Comment (mjsax) [inline docs/customer-360/aviation/process.sql:66]: formatting

File: docs/customer-360/aviation/process.sql [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+                            , UPDATED_DEP TIMESTAMP
+                            , REASON      VARCHAR
+                             )
+              WITH (KAFKA_TOPIC='flight_updates'
+                   , FORMAT='AVRO'
+                   , PARTITIONS=6);
+
+CREATE STREAM customer_flight_updates AS
+  SELECT  CUSTOMER_NAME
+      , FU.REASON      AS FLIGHT_CHANGE_REASON
```
Comment (mjsax) [inline docs/customer-360/aviation/process.sql:76]: and here

File: docs/customer-360/aviation/process.sql (hunk 1 above)
Comment (ybyzek) [reply docs/customer-360/aviation/process.sql:4]: Formatting concerns are covered as part of https://github.com/confluentinc/ksqldb-recipes/issues/90

File: docs/customer-360/aviation/process.sql (hunk 2 above)
Comment (ybyzek) [reply docs/customer-360/aviation/process.sql:13]: Formatting concerns for this recipe are covered as part of https://github.com/confluentinc/ksqldb-recipes/issues/90 (cc: @rmoff )

**Pull request merged**